app/views/govuk_publishing_components/components/docs/metadata.yml

```diff
@@ -26,6 +26,9 @@ examples:
       first_published: 14 June 2014
       last_updated: 10 September 2015
   updated_with_links_to_see_details:
+    description: If a `last_updated` value exists and `see_updates_link` is set to true,
+      a "See all updates" link will be shown. The link's href will be set to `full-publication-update-history`,
```

Comment:
Nitpick: should the href be `#full-publication-update-history`?

Reply:
Thanks @AshGDS, good spot. I'll update it